SU-SOE/stanford_soe_helper
# Add Link to Collections Titles

# READY FOR REVIEW

# Summary
This includes code that makes the collection title in the articles collection top block link and have link color selectable in the collection.

# Needed By (Date)
- Sprint end

# Criticality
- Tweaks things


# Steps to Test
- Switch to this branch  -  7.x-2.x-dev-SOE-2826 (https://github.com/SU-SOE/stanford_soe_helper/tree/7.x-2.x-dev-SOE-2826)
- drush fr stanford_soe_helper_collection and drush fr stanford_soe_helper_collection_views
- drush cc all
- Go edit a collection and there is now a "Accent color" field where you can select the link accent color for the collection to show on the top block in an article for the collection
- Select a color for the collection
- View any article in the collection and verify the accent color shows correctly for the "Article Collection Top Block" collection title.

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2826

## Related PRs

## More Information

## Folks to notify
@cjwest 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
